compiler/src/dmd/funcsem.d

```diff
@@ -1217,3 +1217,152 @@ extern (D) bool checkForwardRef(FuncDeclaration fd, const ref Loc loc)
     }
     return false;
 }
+
+/*************************************************
+ * Find index of function in vtbl[0..length] that
+ * this function overrides.
+ * Prefer an exact match to a covariant one.
+ * Params:
+ *      vtbl     = vtable to use
+ * Returns:
+ *      -1      didn't find one
+ *      -2      can't determine because of forward references
+ */
+int findVtblIndex(FuncDeclaration fd, Dsymbol[] vtbl)
```

Comment:
Missing extern(C++)

Reply:
Say that, wait. I should check whether this is actually used by the C++ backends first.

Reply:
It's unused,leave as extern(D)